autogen/main/README.md

```diff
@@ -263,6 +263,7 @@ The node_pools variable takes the following parameters:
 | queued_provisioning | Makes nodes obtainable through the ProvisioningRequest API exclusively. | | Optional |
 | gpu_sharing_strategy | The type of GPU sharing strategy to enable on the GPU node. Accepted values are: "TIME_SHARING" and "MPS". | | Optional |
 | max_shared_clients_per_gpu | The maximum number of containers that can share a GPU. | | Optional |
+| total_egress_bandwidth_tier |  Specifies the total network bandwidth tier. |  | Optional |
```

Comment:
I believe the accepted values are "TIER_1" and "TIER_UNSPECIFIED".  Let's include in the description similar to the other parameters.

Reply:
will check, its not reflected in provider docs

Reply:
Yeah, I pulled those from here:
- https://github.com/hashicorp/terraform-provider-google/blob/main/google/services/container/resource_container_node_pool_test.go#L696
- https://github.com/hashicorp/terraform-provider-google/blob/main/google/services/container/resource_container_node_pool_test.go#L716